dev/com.ibm.ws.grpc.server/bnd.bnd

```diff
@@ -0,0 +1,57 @@
+#*******************************************************************************
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+Bundle-Name: gRPC Server 1.0.
+Bundle-SymbolicName: com.ibm.ws.grpc.server.1.0
+Bundle-Description: gRPC Server, version 1.0
+
+-dsannotations: \
+  com.ibm.ws.grpc.GrpcApplicationManager
+
+Import-Package: !sun.*,\
+  !com.google.code.gson,\
+  !org.checkerframework,\
+  *
+
+Export-Package: com.ibm.ws.grpc
+
+Require-Capability: osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=11))"
+
+instrument.disabled: true
+
+-buildpath: \
+  com.ibm.ws.container.service;version=latest,\
+  com.ibm.websphere.org.osgi.core;version=latest,\
+  com.ibm.websphere.org.osgi.service.component;version=latest,\
+  com.ibm.wsspi.org.osgi.service.component.annotations;version=latest,\
+  com.ibm.ws.kernel.boot;version=latest,\
+  com.ibm.ws.kernel.service;version=latest,\
+  com.ibm.ws.adaptable.module;version=latest,\
+  com.ibm.ws.app.manager.lifecycle;version=latest,\
+  com.ibm.ws.resource;version=latest,\
+  com.ibm.ws.anno;version=latest,\
+  com.ibm.ws.logging.core,\
+  com.ibm.websphere.javaee.cdi.2.0;version=latest,\
+  com.ibm.websphere.javaee.servlet.4.0;version=latest,\
+  com.ibm.ws.grpc.common.1.0;version=latest,\
+  com.google.code.findbugs:jsr305;version=3.0.2,\
+  com.google.guava:guava;version=27.0.1, \
+  com.ibm.ws.channelfw;version=latest,\
+  com.ibm.ws.transport.http;version=latest,\
+  com.ibm.ws.kernel.service
+  
+javac.source: 1.8
+javac.target: 1.8
```

Comment:
Does this make sense given the java11 requirement?

Reply:
We only need java 8 at build time, so this does make sense